tests/testcases/manual.json

```diff
@@ -595,12 +595,12 @@
     "name": "delegationExpert",
     "tx": {
       "account_number": "6571",
-      "chain_id": "cosmoshub-4",
+      "chain_id": "panacea-3",
       "fee": {
         "amount": [
           {
             "amount": "5000",
```

Comment:
So is this
```suggestion
            "amount": "5",
```

Reply:
We don't need to change this value.
https://github.com/medibloc/ledger-panacea/pull/1#discussion_r853955886

Reply:
Actually, we also don't need to change the chain ID from cosmoshub-4 to panacea-3.